ibis/expr/types/relations.py

```diff
@@ -4619,9 +4625,8 @@ def value_counts(self) -> ir.Table:
         └───────┴───────────┴───────────────────┘
         """
         columns = self.columns
-        return self.group_by(columns).agg(
-            lambda t: t.count().name("_".join(columns) + "_count")
-        )
+        name = name or "_".join(columns) + "_count"
```

Comment:
```suggestion
        if name is None:
        	name = "_".join(columns) + "_count"
```

I always find this self-assignment pattern awkward to read.